main.tf

```diff
@@ -11,6 +11,9 @@ locals {
   ipv4_cidr_block_associations = local.enabled ? (
     length(local.additional_cidr_blocks_map) > 0 ? local.additional_cidr_blocks_map : var.ipv4_additional_cidr_block_associations
   ) : {}
+  default_adoption_tags = merge(module.label.tags, {
+    Name = join(module.label.delimiter, [module.label.id, "default"])
```

Comment:
@aknysh  The more I look at this, the more this seems like an `attribute`. I wonder if we should instantiate the null label module again.